reana_client/cli/quotas.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA client quotas related commands."""
+
+import logging
+import sys
+
+import click
+from reana_client.cli.utils import (
+    add_access_token_options,
+    check_connection,
+    NotRequiredIf,
+)
+
+health_color = {"healthy": "green", "warning": "yellow", "critical": "red"}
+
+
+def usage_percentage(usage, limit):
+    """Usage percentage."""
+    if limit == 0:
+        return ""
+    return "({:.1%})".format(float(usage) / float(limit))
```

Comment:
In the UI we decided [not to go for decimals](https://github.com/reanahub/reana-ui/pull/143/files#diff-f31f6c265fb3528a1f2222e92ee2c8c95621a83c5f90543f95a97b1bb0550548R29) in the end to simplify the output. We can be consistent here.